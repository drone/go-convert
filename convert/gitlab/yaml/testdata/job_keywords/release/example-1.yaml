# https://docs.gitlab.com/ee/ci/yaml/#release

release_job:
  stage: release
  image: registry.gitlab.com/gitlab-org/release-cli:latest
  rules:
    - if: $CI_COMMIT_TAG                  # Run this job when a tag is created manually
  script:
    - echo "Running the release job."
  release:
    tag_name: $CI_COMMIT_TAG
    name: 'Release $CI_COMMIT_TAG'
    description: 'Release created using the release-cli.'