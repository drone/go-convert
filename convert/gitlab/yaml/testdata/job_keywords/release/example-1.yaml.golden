release_job:
    image:
        name: registry.gitlab.com/gitlab-org/release-cli:latest
    release:
        tag_name: $CI_COMMIT_TAG
        name: Release $CI_COMMIT_TAG
        description: Release created using the release-cli.
    rules:
        - if: $CI_COMMIT_TAG
    script:
        - echo "Running the release job."
    stage: release
